app/(header-compte)/(compte)/mes-equipes/page.tsx

```diff
@@ -0,0 +1,39 @@
+import { getRoles } from '#clients/api-d-roles';
+import { GroupManagement } from '#components/espace-agent-components/group-management';
+import { Groups } from '#models/groups';
+import getSession from '#utils/server-side-helper/app/get-session';
+import { Metadata } from 'next';
+
+export const metadata: Metadata = {
+  title: 'Vos équipes de l’Annuaire des Entreprises',
+  alternates: {
+    canonical: 'https://annuaire-entreprises.data.gouv.fr/mes-equipes',
+  },
+  robots: 'noindex, nofollow',
+};
+
+const MesEquipesPage = async () => {
+  const session = await getSession();
+
+  const [roles, groups] = await Promise.all([
+    getRoles(),
```

Comment:
Calling `getRoles` every time can be inefficient unless you cache it in some way

Plus that's a shame to directly call client here. If I were you I would have some basic cache logic in model.

Currently here is what you got : 
``` in models/
groups
group
```

I advise you merge these into a single folder, maybe move this folder inside `/authentication`as they are related : 

```
authentication
  agent/
  user/
  group/
	index.ts
	groups.ts
	roles.ts --> you can use DataStore to have some basic cache 
```

NB: group folder could also be moved into agent folder